package.json

```diff
@@ -51,7 +51,7 @@
     "@babel/plugin-syntax-do-expressions": "^7.24.7",
     "@babel/plugin-syntax-function-bind": "^7.24.7",
     "@babel/preset-react": "^7.24.7",
-    "@types/eslint": "=7.2.10",
+    "@types/eslint": "=8.56.10",
```

Comment:
this is done intentionally, since we support older eslint versions - also what about the estree types?

Reply:
I wasn't aware that the support for older ESLint versions was intentional, and I hadn't considered the estree types either. Given this, I'll go ahead and close the PR for now.
Apologies for the confusion 🙇‍♂️